ddtrace/internal/core/__init__.py

```diff
@@ -124,10 +126,6 @@ def _on_jsonify_context_started_flask(ctx):
 from .event_hub import reset as reset_listeners  # noqa:F401
 
 
-if TYPE_CHECKING:
```

Comment:
This is important for avoiding circular imports. I don't see any non-annotation uses of `Span`, so I expect that keeping this conditional will not break any other parts of your change.

Reply:
Annotations are supposed to be always defined. You can't not import them. But typings module should never create circular dependencies.